tests/components/reolink/test_init.py

```diff
@@ -180,26 +180,6 @@ async def test_credential_error_three(
     assert (HOMEASSISTANT_DOMAIN, issue_id) in issue_registry.issues
 
 
-async def test_entry_reloading(
```

Comment:
Why would this test fail after the changes in this PR?

The options flow should still reload the integration right?

Reply:
Yes, but there's no option flow in this test. It was only manually updating the config entry so it would trigger an update listener (which I have now removed). So this test does not make sense anymore.

Reply:
Right, thanks for the clarification 👍